### PR TITLE
Accounts updated

### PR DESCRIPTION
**Pull request by jsflca2012**
@NeilAllavarpu 
fixed spacing and sorts alphabetically. looks a lot better on responsive views.

### CODE REVIEW

File: src/components/FeatureViews/Accounts/Accounts.js [hunk 1]
```diff
@@ -99,22 +102,28 @@ class Accounts extends Component {
...
+            () => {
                 sessionStorage.setItem('AccountsState', JSON.stringify(this.state));
             }
         );
     }
 
     render() {
+        console.log(Object.values(this.state.usersList).sort(function(a, b) {
+            return (a.name < b.name) ? -1 : (a.name > b.name) ? 1 : 0;
+        }));
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/Accounts.js]: Remove `console.log`s when finished.

File: src/components/FeatureViews/Accounts/ProfileHeading.js [hunk 2]
```diff
@@ -222,20 +222,23 @@ class ProfileHeading extends Component {
         return (
             <div>
                 <Grid container item xs={12} alignItems="center">
-                    <Grid item xs={6}>
+                    <Grid item xs={9} md={9} align="left">
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/ProfileHeading.js]: Repetitive `md` property

**Comment by jsflca2012** [comment]
@NeilAllavarpu fixed your changes, still a bug with the tab on the parent contact redirect though, tabs not switching back to first tab after redirect.

File: src/components/FeatureViews/Accounts/TabComponents/CourseViewer.js [hunk 3]
```diff
@@ -89,15 +89,15 @@ class CourseViewer extends Component {
...
-            }
-        });
-        return numPaidEnrollments;
+        // let courseEnrollment = this.state.userEnrollments[courseID],
+        //     enrollmentPayments = Object.values(courseEnrollment.session_payment_status),
+        //     numPaidEnrollments = 0;
+        // enrollmentPayments.forEach((paymentStatus) => {
+        //     if (paymentStatus === 1) {
+        //         numPaidEnrollments++;
+        //     }
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/CourseViewer.js]: Why was this commented out?

Comment (jsflca2012) [reply src/components/FeatureViews/Accounts/TabComponents/CourseViewer.js]: It kept breaking my app whenever I went onto student profiles.

**Comment by jsflca2012** [comment]
@NeilAllavarpu added a few more changes for navigation bug and backbutton responsive view

**Pull request merged**